lib/extendedQueries.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash/fp')
+
+const concatSchema = (qry, tables) => {
+  const schemaName = 'public' // fetch schema name from Async Local Storage here
```

Comment:
Is this comment necessary?

Reply:
well not really no :)
it was just so I could remember why that variable exists if I forgot, before we implement ALS